src/hooks/useSortedRoutesWithQuotes.ts

```diff
@@ -106,6 +106,8 @@ export const useSortedRoutesWithQuotes = ({
 
   // Only routes with quotes are sorted.
   const sortedRoutesWithQuotes = useMemo(() => {
+    const sortPriority = config.ui.routeSortPriority || 'fastest';
```

Comment:
The previous behavior was to sort by fastest, so we default to fastest when the config is undefined.